ar-octopus.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rake'
   s.add_development_dependency 'rspec', '>= 3'
   s.add_development_dependency 'rubocop'
-  s.add_development_dependency 'sqlite3', '>= 1.3.4'
+  s.add_development_dependency 'sqlite3', '~> 1.3.6'
```

Comment:
Fix for  https://travis-ci.org/thiagopradi/octopus/builds/501538618?utm_source=github_status&utm_medium=notification

This is fixed in Rails master (rails/rails#35154)
but still necessary for older Rails versions.